pkg/cmd/roachprod/vm/aws/BUILD.bazel

```diff
@@ -23,3 +24,25 @@ go_library(
         "@org_golang_x_time//rate",
     ],
 )
+
+# Checks: go_library is only expecting .go files as sources.
```

Comment:
I don't really know what this comment is supposed to mean. What are you trying to express here?

Reply:
That I am adding this new `main.tf` file but I cannot create a dependency within the main `go_library` as `go_library` is only expecting `.go` files and not `.tf`

Reply:
Well, the `genrule` is unused right now, so did you mean to use it somewhere?

Reply:
Yes it is not used yet, I will include it in the next batch of `go:generate` files (mostly go generics)